Dockerfile

```diff
@@ -2,22 +2,23 @@ FROM alpine:3.6 AS wget
 RUN apk add --no-cache ca-certificates wget tar
 
 FROM wget AS docker
-ARG DOCKER_VERSION=17.04.0-ce
-RUN wget -qO- https://get.docker.com/builds/Linux/x86_64/docker-${DOCKER_VERSION}.tgz | \
+ARG DOCKER_VERSION=17.09.0-ce
+RUN wget -qO- https://download.docker.com/linux/static/stable/x86_64/docker-${DOCKER_VERSION}.tgz | \
```

Comment:
You could even skip this entire build-stage; the `COPY --from` syntax also allows copying from an image. The following would pulls the official `docker:17.09` image, and copy the CLI binary from that;

```Dockerfile
COPY --from=docker:17.09 /usr/local/bin/docker /bin
```